includes/abstracts/actions/edit-renewal.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace AutomateWoo_Subscriptions;
+
+defined( 'ABSPATH' ) || exit;
+
+
+/**
+ * Define shared methods to edit renewal date on a subscription.
+ *
+ * @class Abstract_Action_Subscription_Edit_Renewal
+ * @since 1.0
+ */
+abstract class Abstract_Action_Subscription_Edit_Renewal extends \AutomateWoo\Action {
```

Comment:
```suggestion
abstract class Abstract_Action_Subscription extends \AutomateWoo\Action {
```

Filename can be: `subscription.php`